File: others/perl/fizzbuzz.pl
#!/usr/bin/env perl6

my @n = 1..100;

for @n -> $num {
  if $num %% 15 {
    print "Fizzbuzz, ";
  } elsif $num %% 5 {
    print "Buzz, ";
  } elsif $num %% 3 {
    print "Fizz, ";
  } else {
    print "$num. ";
  }
}

File: others/ruby/fizzbuzz.rb
#!/usr/bin/env ruby

(1..100).each do |i|
  if i % 15 == 0
    print "FizzBuzz"
  elsif i % 5 == 0
    print "Buzz"
  elsif i % 3 == 0
    print "Fizz"
  else
    print i.to_s
  end
  print ","
end
